priv/templates/home.tpl

```diff
@@ -15,7 +15,7 @@
 				<h2 class="bordered-title">Ook Interessant</h2>
 			</div>
 
-			{% if m.kc_user.recommended_knowledge_groups as items  %}
+			{% if m.kc_user.recommended_knowledge_groups|slice:[,4] as items  %}
```

Comment:
Instead of limiting it here, it would perhaps be better to do it in the model, so that the search itself stops at 4 results.

You can change the implementation of the `recommended_knowledge_groups` function in `m_kc_user` to use a pagesize of 4 (instead of the current 20).